fastlane/lib/fastlane/actions/reset_simulator_contents.rb

```diff
@@ -42,9 +40,19 @@ def self.description
       def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :ios,
+                                       deprecated: "Use `:os_versions` instead",
                                        short_option: "-i",
                                        env_name: "FASTLANE_RESET_SIMULATOR_VERSIONS",
-                                       description: "Which versions of Simulators you want to reset content and settings, this does not remove/recreate the simulators",
+                                       description: "Which OS versions of Simulators you want to reset content and settings, this does not remove/recreate the simulators",
+                                       is_string: false,
+                                       optional: true,
+                                       type: Array,
+                                       default_value: ENV["FASTLANE_RESET_SIMULATOR_OS_VERSIONS"],
```

Comment:
Do we need this?

Reply:
I don't think we need this since this option is getting deprecated. If somebody has `FASTLANE_RESET_SIMULATOR_OS_VERSIONS` set, they are probably trying to use `os_versions` and not `ios`so I think we can 💣 this

Reply:
I don't think we need this since this option is getting deprecated. If somebody has FASTLANE_RESET_SIMULATOR_OS_VERSIONS set, they are probably trying to use `os_versions` and not `ios` I think we can 💣 this